<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RiivolutionPatchPage</class>
 <widget class="QWidget" name="RiivolutionPatchPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="title" stdset="0">
   <string>Параметры создания патча для Riivolution</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QRadioButton" name="useDisc">
     <property name="text">
      <string>Использовать диск с игрой</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">sourceGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="useImage">
     <property name="text">
      <string>Использовать бэкап-образ</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">sourceGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="sourceTypes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Привод с диском игры:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cdList"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Путь к файлу бэкап-образа:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="imagePath">
           <property name="placeholderText">
            <string>Укажите путь к файлу бэкап-образа...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="selectImageButton">
           <property name="toolTip">
            <string>Выбрать файл бэкап-образа</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="generateToDrive">
     <property name="text">
      <string>Создать патч на накопителе</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">destGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="generateToDirectory">
     <property name="text">
      <string>Создать патч в указанной директории</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">destGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="destTypes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Накопитель для создания патча:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="usbList"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Путь к директории для создания патча:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLineEdit" name="destPath">
           <property name="placeholderText">
            <string>Укажите путь к файлу бэкап-образа...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="selectImageButton_2">
           <property name="toolTip">
            <string>Выбрать файл бэкап-образа</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="destGroup"/>
  <buttongroup name="sourceGroup"/>
 </buttongroups>
</ui>
